smart_campus/app/forms.py

```diff
@@ -19,3 +19,15 @@ def __init__(self, *args, **kwargs):
     class Meta:
         model = Station
         fields = ('name', 'content', 'category')
+
+
+class CategoryForm(forms.ModelForm):
```

Comment:
`StationCategoryForm`
Explicit is better than inexplicit.

Reply:
okay